src/main/resources/crafter/studio/database/upgrade-4.0.1.3-to-4.0.1.4.sql

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2007-2022 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+-- Delete orphan audit_parameters records from deleted sites
+DELETE ap.*
```

Comment:
Double-check the need for `ap`.

Perhaps:
`DELETE FROM audit_parameters WHERE audit_id ...`